src/dbnode/storage/database_test.go

```diff
@@ -525,6 +525,63 @@ func TestDatabaseBootstrappedAssignShardSet(t *testing.T) {
 	wg.Wait()
 }
 
+func TestDatabaseAssignShardSetDuringBootstrap(t *testing.T) {
```

Comment:
Do we also need a similar test for `UpdateOwnedNamespaces`?

Reply:
Added in 13b1cdcae1fb9b6cc591b3a6a176c6945c129c68